protobuf/CMakeLists.txt

```diff
@@ -6,15 +6,17 @@ ExternalProject_Add(
         URL "https://github.com/protocolbuffers/protobuf/releases/download/v3.11.2/protobuf-cpp-3.11.2.tar.gz"
         URL_MD5 "351a0f5c4edd3f45035417f09133ce73"
         CONFIGURE_COMMAND ${CMAKE_COMMAND} -DCMAKE_INSTALL_PREFIX=${protobuf_PREFIX}
-            -DCMAKE_BUILD_TYPE=Release -Dprotobuf_BUILD_TESTS=OFF
+            -DCMAKE_BUILD_TYPE=Release -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_MSVC_STATIC_RUNTIME=OFF
```

Comment:
I guess I cannot, accept it, as this will not work on ubuntu I guess...

Reply:
okay, I'll change it to be platform dependent